internal/sdkv2provider/resource_cloudflare_record.go

```diff
@@ -306,7 +306,7 @@ func resourceCloudflareRecordUpdate(ctx context.Context, d *schema.ResourceData,
 	}
 
 	if comment, ok := d.GetOk("comment"); ok {
-		updateRecord.Comment = comment.(string)
+		updateRecord.Comment = comment.(*string)
```

Comment:
this won't work. it needs to be `cloudflare.StringPtr(comment.(string))`

Reply:
Word. Fixed!